youtube_transcript_api/formatters.py

```diff
@@ -134,13 +134,66 @@ def format_transcripts(self, transcripts, **kwargs):
         """
         return '\n\n\n'.join([self.format_transcript(transcript, **kwargs) for transcript in transcripts])
 
+class SRTFormatter(Formatter):
+    def _seconds_to_timestamp(self, time):
+        """Helper that converts `time` into a transcript cue timestamp for SRT.
+
+        :param time: a float representing time in seconds.
+        :type time: float
+        :return: a string formatted as a cue timestamp, 'HH:MM:SS,MS'
+        :rtype str
+        :example:
+        >>> self._seconds_to_timestamp(6.93)
+        '00:00:06,930'
+        """
+        time = float(time)
+        hours, remainder = divmod(time, 3600)
+        mins, secs = divmod(remainder, 60)
+        ms = int(round((time - int(time))*1000, 2))
+        return "{:02.0f}:{:02.0f}:{:02.0f},{:03d}".format(hours, mins, secs, ms)
```

Comment:
If applying the above fix the format string will have to use `d` instead of `f`.